src/frequenz/client/dispatch/test/_service.py

```diff
@@ -35,6 +35,12 @@
 from .._internal_types import DispatchCreateRequest
 from ..types import Dispatch
 
+ALL_KEY = "all"
+"""Key that has access to all resources."""
+
+NONE_KEY = "none"
+"""Key that has no access to any resources."""
```

Comment:
So, I guess this is only for development? I think it still needs some clarification, as it is it makes no sense, this is a public repo so everyone knows they can use "all" to have full authorization.

Reply:
defined in a file that creates a "fake service" created by a "fake client" in a package called "test" .. come on..

Reply:
![chill](https://c.tenor.com/ZushQNXwhF0AAAAC/tenor.gif)

I didn't see the file name, sorry about that.